assets/app/scripts/services/metrics.js

```diff
@@ -69,7 +74,9 @@ angular.module("openshiftConsole")
           // Save the raw value before we change it.
           value = point.value;
           point.value = millicoresUsed(point, lastValue);
-          lastValue = value;
+          if (value) {
+            lastValue = value;
```

Comment:
I believe we should be able to use the previous good value in the next calculation since the usage tracked is cumulative even if there's a gap where no data is collected. So it's intentional. But I'm wondering if we shouldn't skip the one data point

I'll change this